shared/src/ledger/eth_bridge/bridge_pool_vp.rs

```diff
@@ -117,11 +127,11 @@ where
         // but that will be a separate PR.
         let pending_key = get_pending_key();
         let pending_pre: HashSet<PendingTransfer> =
-            self.ctx.read_pre_value(&pending_key).ok_or(eyre!(
+            (&self.ctx).read_pre_value(&pending_key)?.ok_or(eyre!(
```

Comment:
hm why do we need the explicit borrow here, on `ctx`? was the compiler (or clippy) complaining about something?

Reply:
It's because the trait this method comes from is implemented only for `&Ctx` and not `Ctx`

https://github.com/anoma/namada/blob/635f68cbf39656c2390d784f28104a98e57ed991/shared/src/ledger/native_vp.rs#L337

If there is a nice way to implement for both `Ctx` and `&Ctx` without too much duplication, I'd like to add it (though not in this PR) for ergonomics. (This `StorageReader` thing actually needs to be contributed back to `main` or may even be able to be replaced by the new native storage traits coming in soon)